server/app/services/applications/PdfExporterService.java

```diff
@@ -28,10 +28,15 @@ public final class PdfExporterService {
    * to review applications.
    */
   public PdfExporter.InMemoryPdf generateApplicationPdf(
-      ApplicationModel application, boolean showEligibilityText, boolean includeHiddenBlocks) {
+      ApplicationModel application,
+      boolean showEligibilityText,
```

Comment:
Devin, showEligibilityText --> is also set for only admins. I think we can rename this variable to isAdmin and make it as one variable.